code/modules/admin/topic.dm

```diff
@@ -36,6 +43,7 @@
 
 		message_admins("[key_name_admin(usr)] marked [C.key]'s admin help as an IC issue.")
 		log_admin_private("[key_name(usr)] marked [C.key]'s admin help as an IC issue.")
+		spamcooldown = world.time + 300 // 30 seconds

```

Comment:
is there a way to affect a variable in this datum when an ahelp comes in, I tried some things but nothing seemed to work